src/core/utils/bounding-box.ts

```diff
@@ -240,15 +240,15 @@ function traverseAndCollectTrackInfo(
                         x: dx,
                         y: dy + cumHeight,
                         width: cumWidth,
-                        height: track.height
+                        height: track.height!
                     },
                     layout: { x: 0, y: 0, w: 0, h: 0 } // Just put a dummy info here, this should be added after entire bounding box has been determined
                 });
 
                 if (array[i + 1] && array[i + 1].overlayOnPreviousTrack) {
                     // do not add a height
                 } else {
-                    cumHeight += track.height;
+                    cumHeight += track.height!;
```

Comment:
many many non-null assertions that from the code in this context I cannot tell are "safe".